lib/pages/main/main_vc.dart

```diff
@@ -95,362 +49,219 @@ class _MainVCState extends State<MainVC> {
     });
   }
 
+  void getPackageInfo() async {
+    var _packageInfo = await PackageInfo.fromPlatform();
+    setState(() {
+      packageInfo = _packageInfo;
+    });
+  }
+
+  static List<Widget> _widgetOptions = <Widget>[
+    HomePage(),
+    MyReportsPage(),
+    MyReportsPage(),  // TODO: Public map
+    TutorialPage(true),
+    SettingsPage(),  // TODO: Create new settings page
+    SettingsPage(),  // TODO: Create info page
+  ];
+
+  void _onItemTapped(int index) {
+    setState(() {
+      _selectedIndex = index;
+    });
+  }
+
   @override
   Widget build(BuildContext context) {
-    return Stack(
-      children: <Widget>[
-        Scaffold(
-            appBar: AppBar(
-              backgroundColor: Colors.white,
-              centerTitle: true,
-              leading: IconButton(
-                icon: Icon(
-                  Icons.settings,
-                ),
-                onPressed: () {
-                  Navigator.push(
-                    context,
-                    MaterialPageRoute(
-                        builder: (context) =>
-                            SettingsPage()),
-                  );
-                },
-              ),
-              title: Image.asset(
-                'assets/img/ic_logo.png',
-                height: 40,
-              ),
-              actions: <Widget>[
-                IconButton(
-                  icon: SvgPicture.asset(
-                    'assets/img/sendmodule/ic_adn.svg',
-                    height: 26,
+    return Scaffold(
+      appBar: AppBar(
+        backgroundColor: Colors.white,
+        centerTitle: true,
+        title: Image.asset(
+          'assets/img/ic_logo.png',
+          height: 40,
+        ),
+        actions: <Widget>[
+          badges.Badge(
+            position: badges.BadgePosition.topEnd(top: 2, end: 2),
+            showBadge: unreadNotifications > 0,
+            badgeContent: Text('$unreadNotifications', style: TextStyle(color: Colors.white)),
+            child: IconButton(
+              padding: EdgeInsets.only(top: 6),
+              icon: Icon(Icons.notifications, size: 32, ), 
+              onPressed: () {
+                Navigator.push(
+                  context,
+                  MaterialPageRoute(builder: (context) => NotificationsPage(onNotificationUpdate: updateNotificationCount)),
+                );
+              },
+            )
+          )
+        ],
+      ),
```

Comment:
This appbar appears on all parent pages that are accessed through the drawer. In the subsequent pull requests I'll remove the previous appbars from each individual page to avoid duplicated appbars and being ugly visually